src/exporter-utils.ts

```diff
@@ -25,3 +27,21 @@ export function getHoneycombSpanExporter(
   }
   return configureHoneycombHttpProtoTraceExporter(options);
 }
+
+/**
+ * Builds and returns an OTLP Metric reader that configures a metric exporter to send data over http/protobuf periodically
+ * @param options The {@link HoneycombOptions} used to configure the exporter
+ * @returns a {@link PeriodicExportingMetricReader} configured to send telemetry to Honeycomb over http/protobuf
+ */
+export function getHoneycombMetricReader(
+  options?: HoneycombOptions,
+): PeriodicExportingMetricReader | undefined {
+  if (!options?.metricsDataset) {
```

Comment:
doing the check here for two reasons:
- its impossible to test the `NodeSDK` because everything is private on that class
- i like the idea of `opentelemetry-node.ts` not containing any logic other than calling other functions